osclib/conf.py

```diff
@@ -44,6 +44,8 @@
         'openqa': 'https://openqa.opensuse.org',
         'lock': 'openSUSE:%(project)s:Staging',
         'lock-ns': 'openSUSE',
+        'delreq-monitor': 'factory-maintainers',
+        'main-repo': 'standard',
```

Comment:
This should likely be used in `repo_checker` as well since for lack of having a proper variable I kept the hard-coded `standard`. I'll update that once this is merged.